doc/dev/tests.md

```diff
@@ -5,22 +5,26 @@ title: Tests
 
 ## Overview
 
-Npgsql has an extensive test suite to guard against regressions. The test suite is run on the official build server for all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
+Npgsql has an extensive test suite to guard against regressions. The test suite is run on the official build server for
+the .NET Framework and .NET Core with all supported PostgreSQL backends.
```

Comment:
Line breaks change nothing (:

Reply:
Just easier in the editor, no intended appearance changes

Reply:
All (most?) the documentation I've written at the moment has no line breaks, as these look pretty bad depending on what the width of your editor window happens to be (different across contributors)... it seems better to avoid line breaks and simply enable line wrapping, no?

Reply:
I was working in some other (non-npgsql) markdown files that had line breaks, so it looked right at the time, but I don't have any real opinion on it, one way or the other. Happy to revert.